src/test/java/org/openstreetmap/atlas/checks/event/MetricFileGeneratorTest.java

```diff
@@ -26,30 +26,30 @@
     private static final int BATCH_SIZE = FileProcessor.BATCH_SIZE;
     private static final MetricEvent SAMPLE_EVENT;
 
-    @Rule
-    public TemporaryFolder tempFolder = new TemporaryFolder();
-
     static
     {
         SAMPLE_EVENT = new MetricEvent("a-metric-name", Duration.ONE_MINUTE);
     }
 
+    @Rule
+    public TemporaryFolder tempFolder = new TemporaryFolder();
+
     @Test
-    public void testFileName() throws IOException
+    public void testFileLabelWithKnownSuffix() throws IOException
     {
-        final File tempDirectory = tempFolder.newFolder();
-        final MetricFileGenerator generator = new MetricFileGenerator("some-metrics-log.csv",
+        final File tempDirectory = this.tempFolder.newFolder();
+        final MetricFileGenerator generator = new MetricFileGenerator("some.metrics.txt",
```

Comment:
Why the change in the filename? I understand it is just a test, just curious as to the reasoning behind it.

Reply:
This is part of the reordering.  Eclipse is reordering all method to be alphabetical.  So it's not a change in filename, it's that the ordering of the tests are different.  The ```testFileName()``` test case is now found on line 62.